### PR TITLE
Merge 1.7.6.x into 1.7.7.x - 21/01/2020

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Merge last changes from 1.7.6.x into 1.7.7.x
| Type?         | improvement
| Category?     | ME
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |  No QA needed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17267)

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -531,7 +533,6 @@ GitHub contributors:
 - Priyank Desai
 - ProBackup-nl
 - Profileo
-- Progi1984
```
Comment (matks) [inline CONTRIBUTORS.md:534]: Was it a duplicate ?

**Review by matks — APPROVED** [review]

File: CONTRIBUTORS.md (hunk 1 above)
Comment (Progi1984) [reply CONTRIBUTORS.md:534]: Affirmative, my real name is indicated...

Comment (matks) [reply CONTRIBUTORS.md:534]: I thought Franck was your nickname 😮 and Progi1984 your real name !

**Comment by matks** [comment]
Thanks @Progi1984

File: CONTRIBUTORS.md (hunk 1 above)
Comment (Progi1984) [reply CONTRIBUTORS.md:534]: Shut up, it's a secret !

**Pull request merged**